src/PowerToken.sol

```diff
@@ -219,10 +219,10 @@ contract PowerToken is IPowerToken, EpochBasedInflationaryVoteToken {
         uint16 currentEpoch_ = _clock();
 
         uint40 timeRemaining_ = _isVotingEpoch(currentEpoch_)
-            ? PureEpochs._EPOCH_PERIOD
```

Comment:
It was prefixed to follow the `private-vars-leading-underscore` solhint rule. But we can disable it if it's redundant.